### ISSUE TITLE
Formatting issue with '\"'

### ISSUE THREAD

**Issue by Gekctek** [dfinity/prettier-plugin-motoko#85]
Whenever i have the '\"' literal in my code, it can compile but I get weird formatting issues
All the comments i have inline in the file freak out and comment out the next line

Here is an example. Just format this file in vs code and it will move the comment to the next line and comment out the let c:
```
import Text "mo:base/Text";
module {
    public func A(){
        let a = '\"';
        let b = Text.fromChar(a); // Convert to text
        let c = 1;
    }
}
```

If i switch the literal to '\u{22}' it works fine

**Comment by rvanasa** [comment]
This should be fixed in version 0.9.3 of the VS Code extension. Cheers!

**Comment by Gekctek** [comment]
So quick. Cheers!
